test/colors/colors.test.js

```diff
@@ -26,6 +26,8 @@ describe('colors related tests', () => {
         const { stderr, stdout, exitCode } = run(__dirname, [
             '--stats=verbose',
             `--config=${resolve(__dirname, './no-stats.webpack.config.js')}`,
+            '--mode',
+            'production',
```

Comment:
I think it will be better to define `mode: production` inside the config.

Reply:
Yep, let's do it

Reply:
Yep, let's do it